lib/mongoid/token.rb

```diff
@@ -20,14 +20,14 @@ def token(*args)
         add_token_field_and_index(options)
         add_token_collision_resolver(options)
         set_token_callbacks(options)
-        
+
         define_custom_finders(options) if options.skip_finders? == false
         override_to_param(options) if options.override_to_param?
       end
 
       private
       def add_token_field_and_index(options)
-        self.field options.field_name, :type => String, :default => nil
+        self.field options.field_name, :type => String, :default => default_value(options)
```

Comment:
Line is too long. [90/80]<br>Use the new Ruby 1.9 hash syntax.<br>Redundant `self` detected.